game.py

```diff
@@ -529,6 +530,16 @@ def main():
     shoot_counter = 0
 
     while running:
+        ship_data = n.send(p1_server_data_resp)
+        try:
+            modules = ship_data['modules']
+            # Modules are using capitals for    
+            engine_power = ship_data['Engines']
+            o2_power = ship_data['O2']
+
+        except:
+            if not type(ship_data) is str:
+                print(ship_data)
```

Comment:
If somehow the data is not a dict and not a string tell us please! Put in a try-except to avoid raising exceptions when first launching the game.py without a phone-client buddy. Allows to still run in singleplayer, but just no one controlling stats.